Documentation/api.md

```diff
@@ -489,7 +489,7 @@ ServiceMonitorSpec contains specification parameters for a ServiceMonitor.
 | targetLabels | TargetLabels transfers labels on the Kubernetes Service onto the target. | []string | false |
 | podTargetLabels | PodTargetLabels transfers labels on the Kubernetes Pod onto the target. | []string | false |
 | endpoints | A list of endpoints allowed as part of this ServiceMonitor. | [][Endpoint](#endpoint) | true |
-| selector | Selector to select Endpoints objects. | [metav1.LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#labelselector-v1-meta) | true |
+| selector | Selector to select Service objects to get Endpoints from. | [metav1.LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#labelselector-v1-meta) | true |
```

Comment:
It's the other way around. The selector selects `Endpoints` objects, and if there is a respective `Service` object to the `Endpoints` object then that will be used for additional metadata. The point is, this will also work for just `Endpoints` objects, the respective `Service` object is great for additional metadata but it will still work without.